### ISSUE TITLE
Cannot Login - Cookie Expired and Program Fails to Run

### ISSUE THREAD

**Issue by Plebiemaannns** [Nandaka/PixivUtil2#632]
### Prerequisites

* [y] Did you read FAQ section?
* [y] Did you test with the latest [releases](https://github.com/Nandaka/PixivUtil2/releases "latest releases") or [commit](https://github.com/Nandaka/PixivUtil2/commits/master "latest commit") ?

### Description

On the last working PixivUtil version (pixivutil20191118), I got this error.
![logins](https://user-images.githubusercontent.com/60653122/73745525-2df23e80-478e-11ea-9343-5bfa193c2455.png)
I did read the FAQ plus https://github.com/Nandaka/PixivUtil2/issues/505/, and pasted my cookie onto the config.ini but the issue still persists.

When trying to run the latest version (pixivutil20200125.-.CNY2020), I got this error instead.
![1112](https://user-images.githubusercontent.com/60653122/73745523-2df23e80-478e-11ea-9a25-4153f0bb8cfe.png)

### Steps to Reproduce

1. I just start both programs.
**Expected behavior:** [Both of the programs run then ask for further user inputs]
**Actual behavior:** [Both programs encountered bugs and closes immediately]

### Versions

You can get this information from executing `PixivUtil2.py --help`.

Last working version on my PC : pixivutil20191118
Other version I tried on my PC : pixivutil20200125.-.CNY2020

**Comment by Nandaka** [comment]
are using proxy? what is your windows version?

**Comment by Plebiemaannns** [comment]
Sorry for the late reply.
I'm running 64-bit Win7. I don't use any proxies or VPNs.
After a bit of tinkering I'll run the program again soon, see if there's any other problem.

**Comment by pxssy** [comment]
I have the same problem, happened just recently as i had used the program on the 18th.

Haven't had this despite using the cookie login for the better part of 2-3months without updating cookie. 

Windows 10, same pixivutil20191118

**Comment by Nandaka** [comment]
ok so far for me using the same cookie. Maybe check if the `useragent` is different with your actual browser to login?

**Comment by pxssy** [comment]
I tried, doesn't work. Not sure why it works for you. I even created a new pixiv account, verified it as well, nothing works.

**Comment by pxssy** [comment]
updated python to 3.8, used the newest version, works alright. I guess it was just old and something new on the site broke the old version.

**Comment by archet-VIII** [comment]
Running into the same thing, also with pixivutil20191118 on a Win8.1 machine. Had been working fine through the end of my last run on the 14th, but started throwing that error on log-in when I tried doing another run a few days ago. It might be worth noting that after that last successful run I visited some pixiv pages in my browser for the first time in a few weeks; don't know if that might've triggered some site change that blocked the downloader or not, but it's only proximate event I have between it working and not working. The site also had a notice about its new page header design when I visited after the error started, which I guess could be related. 

Also, when I checked the cookies on a current pixiv page I found that my login cookie was unchanged, but I couldn't find anything that matched up with the 'refresh_token' listed under it in my config file. No cookie on the page with that name or one close to it, and nothing that matched up to the format of the one I had saved in my config. 

No comment on how a newer version fares, as I actually haven't been able to get a 3.8-based one running yet. But that's a separate issue.
